README.md

```diff
@@ -142,6 +142,16 @@ recognised:
       * legacy
       * usb
       * both
+  * screen.mode=*w*x*h* (EFI framebuffer only)
+    * where *w*x*h* is the preferred screen resolution (e.g. 1024x768)
+  * screen.mode=bios (EFI framebuffer only)
+    * uses the default screen resolution set by the UEFI BIOS
+  * screen.rhs-up (graphics mode only)
+    * rotates the display clockwise by 90 degrees
+  * screen.lhs-up (graphics mode only)
+    * rotates the display anti-clockwiseby 90 degrees
```

Comment:
```s/anti-clockwiseby/counter-clockwise by/```

Reply:
anti-clockwise is correct English.

Reply:
My bad, I didn't come across this variant before. It looks like "anti-clockwise" is more common in en_GB, while I'm really spoiled by en_US… I'm sorry I didn't check it before commenting, thank you for clarification.

A missing space before ``by`` should be fixed anyway, shouldn't it?

Reply:
Yes, missing space fixed, thanks for spotting that.